### ISSUE TITLE
Readonly properties

### ISSUE THREAD

**Issue by rjuzaszek** [Friend-LGA/LGSideMenuController#96]
Why are leftViewPresentationStyle and right one now readonly (v. 1.1.0 from cocoa pods), and how to set them from vc?

**Comment by rjuzaszek** [comment]
i investigated more, and found that cocoa pods version (1.1.0) doesn't contain LGSideMenuSegue and many more files

**Comment by Friend-LGA** [comment]
Yes, because they were added in later releases. Try to use 2.0.9.

**Comment by rjuzaszek** [comment]
ok thx
